pkg/utils/webhook/webhook_suite_test.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
```

Comment:
a test-suite is defined no tests.

In general we should also add tests for the validation webhooks.

I think we could use the controller-runtime test environment for that

Reply:
I added an integration test. For more unit-like tests, we need a new kubebuilder release - the latest stable release doesn't support `AdmissionReview` version `v1` and the alpha releases don't include the necessary binaries (etcd, kubectl, kube-apiserver).